src/torcontrol.cpp

```diff
@@ -368,14 +368,26 @@ void TorController::add_onion_cb(TorControlConnection& _conn, const TorControlRe
     }
 }
 
+static bool AllowOutboundOnion()
+{
+    if (!gArgs.IsArgSet("-onlynet")) {
+        return true;
+    }
+    for (const auto& net : gArgs.GetArgs("-onlynet")) {
```

Comment:
Please consider using standard algorithms instead of hand-rolled loop:
```
#include <algorithm>
...

    const auto only_nets{gArgs.GetArgs("-onlynet")};

    //TODO: C++20: use std::ranges::any_of() instead
    return std::any_of(only_nets.cbegin(), only_nets.cend(), [](const auto &net) {
        return ParseNetwork(net) == NET_ONION;
    });
```

Reply:
Done.